### ISSUE TITLE
ES2016 testing?

### ISSUE THREAD

**Issue by clavecoder** [getify/es-feature-tests#22]
Now that [ES2016](http://www.2ality.com/2016/01/ecmascript-2016.html) is out, are there plans for including these feature in your browser tests?

**Comment by getify** [comment]
yes i want to... 2016 is small and easy to test for... only 2 features
